tests/cross_fw/test_templates/fq_params/fq_params.json

```diff
@@ -16,9 +16,9 @@
     "weights_symmetric_sign_True_per_ch_False_narrow_range_False_hf_range_True": {
         "levels": 256,
         "input_low": -2.0340540409088135,
-        "input_high": 2.0022718596646167,
+        "input_high": 2.002271890640259,
         "output_low": -2.0340540409088135,
-        "output_high": 2.0022718596646167
+        "output_high": 2.002271890640259
```

Comment:
Looks like the difference is less than eps, could you please keep the references as it is?

Reply:
References should be updated only if the code has been updated. The calculation of fq was not changed in any way, but references were updated. That's confusing, sooner or later someone will be confused by this change. Please keep references as it is

Reply:
done